src/Enums/ticketStates.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace RA\Enums;
+
+abstract class TicketStates
+{
+    public const Closed = 0;
+    public const Open = 1;
+    public const Resolved = 2;
```

Comment:
Constants are usually uppercase/snake cased - e.g. `CLOSED`, `OPEN`, `RESOLVED`

Reply:
I'll make the change. Must be my C# habits creeping into my code.